WordPress/gutenberg
# Blocks: Propose a server side rendering implementation

This is also called dynamic blocks and is useful for blocks like embed, comments blocks, ...
This is still in progress but feedback is appreciated.

**Testing instructions**

 - Insert this `anything` to a post content (old editor) and preview this post, you should see your block replaced by `THIS IS AWESOME`
